### PR TITLE
Update Phabricator to v1beta3 example

### PR DESCRIPTION
**Pull request by bcbroussard**
Small part of #6584.

### CODE REVIEW

File: examples/phabricator/v1beta3/phabricator-controller.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+              {
+                "name": "MYSQL_SERVICE_PORT",
+                "value": "3306"
+              },
+              {
+                "name": "MYSQL_PASSWORD",
+                "value": "1234"
+              }
+            ],
+            "imagePullPolicy": "Always"
```
Comment (nikhiljindal) [inline examples/phabricator/v1beta3/phabricator-controller.json]: This is the default value and hence can be deleted to keep the example short and simple.

File: examples/phabricator/README.md [hunk 2]
```diff
@@ -110,34 +132,48 @@ To automate this process and make sure that a proper host is authorized even if
...
+      "metadata": {
+        "labels": {
+          "name": "authenticator"
+        }
+      },
+      "spec": {
+        "containers": [
+          {
             "name": "authenticator",
-            "image": "kubernetes/example-cloudsql-authenticator",
```
Comment (nikhiljindal) [inline examples/phabricator/README.md]: Why are we changing the image?

File: examples/phabricator/README.md [hunk 3]
```diff
@@ -110,34 +132,48 @@ To automate this process and make sure that a proper host is authorized even if
...
+              {
+                "name": "SELECTOR",
+                "value": "name=phabricator"
+              },
+              {
+                "name": "CLOUDSQL_DB",
+                "value": "phabricator-db"
+              }
             ],
             "imagePullPolicy": "Always"
```
Comment (nikhiljindal) [inline examples/phabricator/README.md]: not required. can delete

File: examples/phabricator/README.md (hunk 2 above)
Comment (bcbroussard) [inline examples/phabricator/README.md]: @nikhiljindal - It appears that the readme listed the wrong image. This json matches the actual image value in both the v1beta1 and v1beta3 json files - https://github.com/GoogleCloudPlatform/kubernetes/blob/master/examples/phabricator/authenticator-controller.json

**Comment by bcbroussard** [comment]
@nikhiljindal - updated with your comments addressed.

File: examples/phabricator/README.md [hunk 4]
```diff
@@ -25,34 +25,42 @@ To start Phabricator server use the file `examples/phabricator/phabricator-contr
...
-              {"name": "MYSQL_PASSWORD", "value": "1234"},
+            "image": "fgrzadkowski/example-php-phabricator",
+            "ports": [
+              {
+                "name": "http-server",
+                "containerPort": 80
+              }
             ],
-            "ports": [{"name": "http-server", "containerPort": 80}],
             "imagePullPolicy": "Always"
```
Comment (nikhiljindal) [inline examples/phabricator/README.md]: can delete this.

File: examples/phabricator/README.md [hunk 5]
```diff
@@ -110,34 +118,38 @@ To automate this process and make sure that a proper host is authorized even if
...
+          "name": "authenticator"
         }
       },
-      "labels": { "name": "authenticator" }
+      "spec": {
+        "containers": [
+          {
+            "name": "authenticator",
+            "image": "fgrzadkowski/example-cloudsql-authenticator",
+            "imagePullPolicy": "Always"
```
Comment (nikhiljindal) [inline examples/phabricator/README.md]: can delete

File: examples/phabricator/phabricator-controller.json [hunk 6]
```diff
@@ -1,30 +1,38 @@
...
-              {"name": "MYSQL_SERVICE_PORT", "value": "3306"},
-              {"name": "MYSQL_PASSWORD", "value": "1234"}
+            "ports": [
+              {
+                "name": "http-server",
+                "containerPort": 80
+              }
             ],
-            "ports": [{"name": "http-server", "containerPort": 80}],
             "imagePullPolicy": "Always"
```
Comment (nikhiljindal) [inline examples/phabricator/phabricator-controller.json]: can delete

**Comment by nikhiljindal** [comment]
Thanks! Mostly LGTM with a few nits.
Please squash your commits so that I can merge.

**Comment by bcbroussard** [comment]
Removed the unnecessary `imagePullPolicy` and squashed. I had left them in since they were in the original v1beta1. Should be good to go.

**Comment by nikhiljindal** [comment]
Thanks. Will merge once travis/shippable turns green

**Pull request merged**